examples/tv-casting-app/tv-casting-common/src/CastingServer.cpp

```diff
@@ -168,7 +168,7 @@ chip::Inet::IPAddress * CastingServer::getIpAddressForUDCRequest(chip::Inet::IPA
         if (ipAddresses[i].IsIPv4())
         {
             ipIndexToUse = i;
-            ChipLogProgress(AppServer, "Found iPv4 address at index: %zu", ipIndexToUse);
+            ChipLogProgress(AppServer, "Found iPv4 address at index: %d", (int)ipIndexToUse);
```

Comment:
If we have to cast this to make the compiler happy, should size_ts be cast to something larger, like unsigned longs (%lu), to avoid a risk of narrowing the size?
Secondly, does this compile for the arm (not arm64) version of the Android tv-casting-app?

Reply:
It complies all variants no? 

I figured this was a safe cast since this number will be small and less than size of int.

Reply:
For the record building arm works fine.